app/konnect/org-management/system-accounts.md

```diff
@@ -16,6 +16,12 @@ System accounts offer the following benefits over regular user accounts:
 
 The system account can use a {{site.konnect_short_name}} personal access token (PAT) the same way a [regular {{site.konnect_short_name}} user](/konnect/org-management/users/) can. In addition, the system account can be assigned roles directly or inherit the roles of a [team](/konnect/org-management/teams-and-roles/). As such, a PAT created by a system account inherits the roles assigned to the system account.
 
+## Managed system accounts
+
+Managed system accounts are system accounts whose life cycle is managed by {{site.konnect_short_name}} instead of the user. The `blank` flag in the API denotes this type of system account.
```

Comment:
```suggestion
Managed system accounts are system accounts whose life cycle is managed by {{site.konnect_short_name}} instead of the user. The `konnect_managed: true` flag in the API denotes this type of system account.
```